File: Programmers/Level1/68644-두개뽑아서더하기.py
def solution(numbers):
    answer = []
    
    numbers.sort()
    length = len(numbers)
    
    for i in range(length-1):
        for j in range(i+1, length):
            if numbers[i]+numbers[j] not in answer:
                answer.append(numbers[i]+numbers[j])
    answer.sort()
    
    return answer

File: Programmers/Level1/68644-두개뽑아서더하기.js
function solution(numbers) {
    let answer = [];
    for(let i = 0; i < numbers.length-1; i++) {
        for(let j = i+1; j < numbers.length; j++) {
            const sum = numbers[i] + numbers[j];
            if(!answer.includes(sum)) answer.push(sum);
        }
    }
    answer.sort((x, y) => x-y);
    
    return answer;
}